src/detector/loader.py

```diff
@@ -30,7 +30,7 @@ class Loader:
     # This also works as the **temporary** solution of loading FileDescriptorSet
     # from API definition files that ussers pass in from the command line.
     _CURRENT_DIR = os.getcwd()
-    PROTOC_BINARY = os.path.join(_CURRENT_DIR, "test/tools/protoc")
+    PROTOC_BINARY = "python -m grpc.tools.protoc"
```

Comment:
This will work for now, but would be better if we find a way to call a function here.

Reply:
Talked offline and the function call will be updated in a separate PR, thanks!